perceval/backends/jira.py

```diff
@@ -274,6 +311,14 @@ def get_issues(self, from_date):
                 issues = req.text
                 self.__log_status(start_at, tissues)
 
+    def get_fields(self):
+        """Retrieve all the fields available.  """
+        s = self.__init_session()
```

Comment:
I should rename `s` to `session`. In Python, is `s` is the common name used to store a string.